application/src/main/java/uk/gov/hmcts/dm/service/DocumentContentVersionService.java

```diff
@@ -30,30 +27,19 @@
     @Autowired
     private StoredDocumentRepository storedDocumentRepository;
 
-    @Autowired
-    @Setter
-    private HttpServletResponse response;
-
-    @Setter
-    @Getter
-    private int streamBufferSize = 5000000;
+    private static int STREAM_BUFFER_SIZE = 5000000;
 
```

Comment:
you mean private static final ?  or do you want to make it configurable later?

Reply:
yes and not configurable.